Documentation/WritingDocForExtension/Migrate.rst

```diff
@@ -32,7 +32,8 @@ a migration tool. This tool can be used to automatically create a
 :file:`Documentation/guides.xml` from the information contained in your
 :file:`Documentation/Settings.cfg`.
 
-`Docker <https://docs.docker.com/install/>`__ needs to be installed on your
+`Docker <https://docs.docker.com/install/>`__ (or a drop-in replacement like
+`Podman <https://podman.io>`__) needs to be installed on your
```

Comment:
This may imply for the unexperienced user that the following command may work. Here, tabs will work.

Reply:
I'm sorry, I don't understand? How would you phrase/change it exactly?

It's just about saying "if you installed podman and not [only] docker, you're fine"?

Reply:
"Docker ... or ... Podman ... needs to be installed on your OS ... to work:

Code:  docker run ...

This implies for the unexperienced user: I install (only) Podman and can run the docker command as given in the code example? At least I read it that way.

Reply:
Yes

Reply:
So why no tabs? We do this also for CLI commands (Composer-based/Legacy).

Reply:
I'll try to dig into this see how it looks!👍

Reply:
Example code: https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/_sources/ApiOverview/CommandControllers/Index.rst.txt

Reply:
Was part of via #403 , resolving.